testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-gcs/core-site-overrides-template.xml

```diff
@@ -5,6 +5,11 @@
         <value>false</value>
     </property>
 
+    <property>
```

Comment:
What happened when this parameter was not specified?

Reply:
the `with space` test in GCS failed.